mlrun/config.py

```diff
@@ -796,7 +796,15 @@ def update(self, cfg, skip_errors=False):
         for key, value in cfg.items():
             if hasattr(self, key):
                 if isinstance(value, dict):
-                    getattr(self, key).update(value)
+                    try:
+                        getattr(self, key).update(value)
+                    except AttributeError as exc:
+                        config_value = getattr(self, key)
+                        if not isinstance(config_value, (dict, Config)):
+                            raise ValueError(
+                                f"Can not update `{key}` config. "
+                                f"Expected a configuration but received {type(config_value)}"
```

Comment:
IIUC what you received is `value` and not `config_value`
```suggestion
                                f"Expected a configuration but received {type(value)}"
```

Reply:
`value` is what you expect it to be, `config_value` is what you have received